zos_concepts/certificate_management/inventories/inventory.yml

```diff
@@ -0,0 +1,17 @@
+################################################################################
+# © Copyright IBM Corporation 2020, 2021
```

Comment:
i have no idea why this file showed up in the diff, it is unchanged since last year, probably worth a year update in any case
<img width="986" alt="image" src="https://github.com/user-attachments/assets/ebbedad6-3f13-4a5e-825e-abad74cdec02">